### PR TITLE
Update chart repo in readme

### PR DESCRIPTION
**Pull request by heubeck**

### CODE REVIEW

**Review by alpeb — COMMENTED** [review]
Thank you @heubeck for the contribution. As you point out, the location of the linkerd-smi chart has changed. However, can you please also leave the instructions for installing the control plane?

File: charts/linkerd-smi/README.md.gotmpl [hunk 1]
```diff
@@ -18,25 +18,22 @@ docs][linkerd-docs].
 ## Prerequisite: Linkerd Core Control-Plane
 
 Before installing the Linkerd SMI extension, The core control-plane has to
-be installed first by following the [Linkerd Install
-Guide](https://linkerd.io/2/tasks/install/).
+be installed first by following the [Linkerd Install Guide](https://linkerd.io/2/tasks/install/)
+or using its [Helm chart](https://github.com/linkerd/linkerd2/tree/main/charts/linkerd-control-plane).
```
Comment (alpeb) [inline charts/linkerd-smi/README.md.gotmpl]: The install guide already has instructions for installing via the Helm method, so there's no need to modify this paragraph.

Comment (heubeck) [reply charts/linkerd-smi/README.md.gotmpl]: Just reacted to your 

> However, can you please also leave the instructions for installing the control plane?

what did you have in mind than?

Comment (alpeb) [reply charts/linkerd-smi/README.md.gotmpl]: I believe you had deleted that paragraph entirely.

Comment (heubeck) [reply charts/linkerd-smi/README.md.gotmpl]: no :smile: 
so rolling back the helm chart ref.

Comment (alpeb) [reply charts/linkerd-smi/README.md.gotmpl]: Sorry, I think I got confused along the way. This looks good to me!

**Review by alpeb — APPROVED** [review]

**Pull request merged**